tests/_fixtures.py

```diff
@@ -67,4 +67,55 @@
      PRIMARY KEY(id, commit_ts DESC);
 """
 
+EMULATOR_DDL = """\
```

Comment:
Is this added because Emulator doesn't support Numeric yet? But it means that if the emulator supports Numeric in the future, we need to remove these code, right?

Reply:
> Is this added because Emulator doesn't support Numeric yet?

Yes. The alternative would be skipping a large number of tests when using the emulator which isn't ideal.

> But it means that if the emulator supports Numeric in the future, we need to remove these code, right?

Potentially. We could also set it to equal `DDL_STATEMENTS` and retain the emulator specific code rather than remove it in case there are features in the future which are not supported by the emulator initially.